experimental/dds/tree2/src/feature-libraries/sequence-field/format.ts

```diff
@@ -3,16 +3,24 @@
  * Licensed under the MIT License.
  */
 
-import { ITreeCursorSynchronous, JsonableTree, RevisionTag } from "../../core";
-import { ChangesetLocalId, NodeChangeset } from "../modular-schema";
+import { TSchema, Type } from "@sinclair/typebox";
+import { ITreeCursorSynchronous, JsonableTree, RevisionTag, RevisionTagSchema } from "../../core";
+import { ChangesetLocalId, ChangesetLocalIdSchema, NodeChangeset } from "../modular-schema";
+
+// TODO:AB#4259 Types in this file are largely re-used for in-memory representation.
+// See for example `Revive` whose type uses ITreeCursorSynchronous, but the schema
+// for the serialized type uses ProtoNode (which is the result of serializing that cursor).
 
 /**
  * The contents of a node to be created
  */
 export type ProtoNode = JsonableTree;
+export const ProtoNode = Type.Any();
```

Comment:
this (and the above reference to JsonableTree) will be switched to be more strict once #15453 is in.

Reply:
that went in, i'll update this pr later today